bindata/network/kuryr/alert-rules.yaml

```diff
@@ -121,3 +121,17 @@ spec:
       for: 5m
       labels:
         severity: critical
+    - alert: KuryrLoadBalancerNotReady
+      annotations:
+        message: Kuryr noticed an Octavia load balancer unexpectedly stuck in PENDING_* state in the last 20 minutes. Most likely this indicates an issue with OpenStack Octavia.
+      expr: |
+        changes(kuryr_load_balancer_readiness_total[20m]) > 0
```

Comment:
The name of the metric differs from what is registered kuryr_load_balancer_readiness

Reply:
Yes, I know :) what is weird, prometheus client produces two child samples for type counter - one with suffix "_total" and one with suffix "_created"[1].

[1] https://github.com/prometheus/client_python/blob/9a24236695c9ad47f9dc537a922a6d1333d8d093/prometheus_client/metrics.py#L276-L280